result/bgpv4.yaml

```diff
@@ -169,4 +173,24 @@ components:
             openconfirm:
               x-field-uid: 5
             established:
-              x-field-uid: 6
\ No newline at end of file
+              x-field-uid: 6
+        gr_restart_attempted:
+          description: |-
+            Number of Graceful Restarts attempted by the neighbor of the BGP peer. This is incremented
+            when a Open Message received indicating that neighbor has restarted.
+          type: integer
+          format: uint32
+          x-field-uid: 17
+        gr_restart_failed:
+          description: |-
+            Number of Graceful Restarts that were attempted by BGP Peer, however the BGP session
```

Comment:
_The number of times that the BGP session could not be re-established within the Graceful Restart time after Graceful Restart was attempted by the neighbor of the BGP peer._ might be better. 
However, confused what happens if OPEN itself is not recieved , attempted stat will not be incremented but this will be ? So we can get _gr_restart_failed_  > _gr_restart_attempted_ ?

Reply:
I dont have the answer, pls let me know want to keep or not

Reply:
Let us keep for now. I guess we need to 
a) check if this theoretical scenario is actually true or not with actual emulated test. 
b)  check for current OC bgp metrics , whether it provides some counters for GR restarts for helper node [ unlikely there will be anything for restarting node ] , and 
c) work out if what have today makes sense or needs some change of behaviour in the imp. for these metric s to make it useful for end-user for validation purposes.  -->  I think having it seems to be better than not having it , even in present form.